microcosm_flask/tests/conventions/fixtures.py

```diff
@@ -123,11 +123,13 @@ def address_delete(address_id, address_clock):
 def address_search(offset, limit, list_param=None, enum_param=None):
     if list_param is None or enum_param is None:
         return [ADDRESS_1], 1
-    return Address(
-        ADDRESS_ID_1,
-        PERSON_ID_1,
-        ",".join(list_param) + str(len(list_param)) + enum_param.value
-    ), 1
+    return [
```

Comment:
This was a weird one to me: the tests assumed that we would auto-convert single objects to lists, but I'm not sure if any service that uses mc-flask used that behavior. This was also breaking, because now the validation was running on the object itself, given the lack of conversion.

Reply:
I don't understand what the `list_param` or `enum_param` args or for. Were these things we thought we wanted to use in microcosm that we never did?

Reply:
As far as I, I think these are just for unit tests. I don't think we have builtin params that support this.